tests/unit/amo/components/TestContribute.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { oneLine } from 'common-tags';
+
+import Contribute from 'amo/components/Contribute';
+import {
+  ADDON_TYPE_EXTENSION,
+  ADDON_TYPE_LANG,
+  ADDON_TYPE_THEME,
+} from 'core/constants';
+import Button from 'ui/components/Button';
+import Card from 'ui/components/Card';
+import { fakeI18n } from 'tests/unit/helpers';
+
+describe(__filename, () => {
+  const render = (props = {}) => {
+    const allProps = {
+      contributionsURL: 'https://paypal.me/babar',
+      addonType: ADDON_TYPE_EXTENSION,
+      numberOfAuthors: 1,
+      i18n: fakeI18n(),
+      ...props,
+    };
+
+    return shallow(<Contribute {...allProps} />, {
+      context: { i18n: allProps.i18n },
+    }).shallow();
```

Comment:
Instead of rendering twice, use `shallowUntilTarget()`. That way it will be a self-maintaining test as we add more HOC decorators.